config/clusters/nasa-veda/common.values.yaml

```diff
@@ -55,11 +50,192 @@ basehub:
             - 2i2c-org:hub-access-for-2i2c-staff
           scope:
             - read:org
+    prePuller:
+      continuous:
+        enabled: true
+      hook:
+        enabled: true
     singleuser:
       defaultUrl: /lab
       image:
+        # NOTE: To have prePuller enabled makes sense as long as we stick with a
+        #       single image option configured from here. If the configurator is
+        #       used, or a list of images is chosen from, then prePuller should
+        #       be disabled to avoid pulling images not to be used, which could
+        #       potentially delay startup of a user pod wanting to pull another
+        #       image.
```

Comment:
I think it's best to have this comment in the `prePuller` section.

Reply:
I think I maybe you are right and i'll relocate it! 

It is a tricky trade off. I made the comment here under singleuser.image and put prePuller section closer. I figure it relates to: prePuller, singleuser.image, profileList where a dropdown of choices may be configured.

Reply:
Resolved by [96399b8](https://github.com/2i2c-org/infrastructure/pull/2340/commits/96399b8db9e8405cec18ba6fa5fefab0b59c54f4)